src/main/scala/amailp/intellij/robot/psi/LibraryValue.scala

```diff
@@ -1,16 +1,21 @@
 package amailp.intellij.robot.psi
 
+import amailp.intellij.robot.ast.{LibraryAlias, LibraryName}
 import amailp.intellij.robot.psi.utils.RobotPsiUtils
 import com.intellij.extapi.psi.ASTWrapperPsiElement
 import com.intellij.lang.ASTNode
 import com.intellij.psi.PsiReference
 import com.intellij.psi.impl.source.resolve.reference.ReferenceProvidersRegistry
+import com.intellij.psi.util.QualifiedName
 
 class LibraryValue(node: ASTNode) extends ASTWrapperPsiElement(node) with RobotPsiUtils with Library {
+  val libraryName: ASTNode = getNode.findChildByType(LibraryName)
```

Comment:
What about putting here after:
`assume(libraryName != null)`

It cannot be `null` because of how is created in `SettingsParser`, but this would make it more explicit here...